qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -78,6 +78,8 @@
                     cpu_model_flags += ",+xsaveerptr,+clzero"
                     flags = "clzero xsaveerptr"
                 - virt-ssbd:
+                    auto_cpu_model = no
+                    cpu_model = EPYC
```

Comment:
Sorry for does not understand the background very well,   except `EPYC-Milan`, what's the expect result for other cpu model, such as  `EPYC-Rome` and may be other new cpu model ?

Reply:
> Sorry for does not understand the background very well, except `EPYC-Milan`, what's the expected result for other cpu model, such as `EPYC-Rome` and may be other new cpu model ?

I can say that this case is for 'flag' test, so we it is not related to special model type too much normally, you can see we set 'auto_cpu_model = yes' in this cfg. 

I put the product bug to the automation bug comment if you want to know more details.

Reply:
But this case, it use `auto_cpu_model = no` and  `cpu_model = EPYC` and with `enforce`(L3).
Yes, i see the product issue already, exclude `EPYC-Milan` is ok for me.  However, what's the expected result for other cpu model ?   
1. only test EPYC , do not test others ?
2. If it is not a EPYC host, skip test, fail test, or anything else ?   

Could you explain a bit more~ Thanks a lot!

Reply:
@yanan-fu update and please review it again, thanks.